src/Makefile

```diff
@@ -64,12 +64,16 @@ build/ftp.js: gen_mira_blob_2.py ../ftp.bin
 build/todex.js: gen_mira_blob_2.py ../todex.bin
 	mkdir -p build
 	python3 gen_mira_blob_2.py ../todex.bin > build/todex.js
+
+build/ps4-hen-vtx.js: gen_mira_blob_2.py ../ps4-hen-vtx.bin
```

Comment:
I'd better place this in src/, not in the root. See how mira.elf et al. are handled. `ftp.bin` is done the other way only because its source code has been lost.

Reply:
i think the source for 6.72 is not released yet , So far only bin has been release

I tried the below
1. download the Source from https://github.com/SiSTR0/ps4-hen-vtx
2. I complied and created the bin file
3. converted it to js
4. triggered after your JB, Nothing happened


git link